lib/rex/post/meterpreter/packet_dispatcher.rb

```diff
@@ -639,6 +638,44 @@ def deregister_inbound_handler(handler)
   attr_accessor :receiver_thread # :nodoc:
   attr_accessor :dispatcher_thread # :nodoc:
   attr_accessor :waiters # :nodoc:
+
+  @tlv_log_file = nil
+
+  def log_packet(packet, packet_type)
+    if framework.datastore['sessiontlvlogging']&.downcase == 'console'
```

Comment:
If we validate this on assignment, we shouldn't need to `downcase` this 🤔 

```suggestion
    if framework.datastore['sessiontlvlogging'] == 'console'
```